lib/workos/directory_user.rb

```diff
@@ -47,6 +47,16 @@ def to_json(*)
       }
     end
 
+    def get_primary_email(*)
```

Comment:
A couple of minor things here 
- why don't we just call this method `primary_email`, I think the `get` can be implicit.
- do we need the splat `(*)` parameter? I don't think we need any parameters for this method? (🤔 I see we have the `*` ☝️ in `to_json` also... is it actually doing something that I'm not thinking of? Or can we just get rid of the splats everywhere?)
```suggestion
    def primary_email
```

Reply:
👍 - yeah I'm not sure of the splat either.